ONSdigital/response-operations-ui
# Allow messages to be close when respondent has been deleted

# What and why?
A rops user should be able to close a conversation even if the respondent has been removed, this PR updates the code base to do this. This was leading to 500 in some situations
# How to test?
This realistically needs access to the DB to test properly as we have no UI to delete a respondent. Create a collection exercise as you normally would and make sure you login to frontstage with a respondent. Start a conversation with them using rops, send a few messages between the 2 making sure it works as normal. Now go to party in the db and find your user and delete the respondent (I changed their UUID so I could toggle without clearing out all the associated tables, which is effectively the same, but did delete as well). Now check you can close the conversation successfully. Also check that once a conversation has been closed it can't be re-opened
# Trello
